docs/guide/global.md

```diff
@@ -0,0 +1,59 @@
+# The global doc
+
+Editable settings used across the Apostrophe app can be managed through the **global doc**. The global doc is a special piece: it is created automatically and is the only doc of its type allowed. It is configured through the `@apostrophecms/global` piece type module.
+
+There are many good uses for the global doc, including configuring:
+
+- company information or navigation in the website footer
+- a special ID for a third-party service, such as Google Analytics
+- social media account URLs
+
+These are all piece of information that are used across many web pages and they may potentially change at some point. They also only need to be defined once or they at least serve as a site-wide default.
+
+To demonstrate, we may want to display an organizational Github URL in the website footer and a "Contact" page. We can add that to the global doc as we would on any other piece type.
+
+```javascript
+// modules/@apostrophecms/global/index.js
+module.exports = {
+  fields: {
+    add: {
+      githubUrl: {
+        type: 'url',
+        label: 'Github organization url'
+      }
+    },
+    group: {
+      basics: {
+        label: 'Basics',
+        fields: ['githubUrl']
+      }
+    }
+  }
+}
+```
+
+::: note
+For normal piece types, we talk about adding fields to the *piece type*. Since there is always only one doc for the "global piece type," we'll often simply talk about configuring *the global doc* instead of *the global piece type*.
```

Comment:
```suggestion
```
I don't think this is necessary. I think its only needed because we're overly defining it as a 'piece'. Its good to know that its technically a piece, but not to the extent where we need notes?